volumes/ecocounter/tables/sites.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE ecocounter.sites (
+    site_id numeric PRIMARY KEY,
+    site_description text NOT NULL,
+    geom geometry(POINT, 4326),
```

Comment:
huhhh im not sure what this fluff error is

Reply:
I think `geometry` is a keyword in standard SQL, not a datatype like in PostGIS